pkg/constants/metadata.go

```diff
@@ -32,6 +32,10 @@ const (
 	// this annotation will be cleared
 	DevWorkspaceStopReasonAnnotation = "controller.devfile.io/stopped-by"
 
+	// DevWorkspaceDebugStartAnnotation enables debugging workspace startup if set to "true". If a workspace with this annotation
+	// fails to start (i.e. enters the "Failed" phase), its deployment will not be scaled down in order to allow viewing logs, etc.
+	DevWorkspaceDebugStartAnnotation = "controller.devfile.io/debug-start"
```

Comment:
Seems it's a good time(or too late) to agree on a convention on annotations/labels names we use.
We have - and _, like `stopped-by` `debug-start` but `endpoint_name`, `devworkspace_name`, `devworkspace_id`.
BTW we need to continue with one format even if leave another existing untouched.

Reply:
+1 to use - for a new annotations and properties.